### PR TITLE
Add support Kubeflow operators to ROADMAP

### PR DESCRIPTION
**Pull request by andreyvelich**
I updated ROADMAP with support MXNet, XGBoost, MPI operator.
Also, I renamed Hyperparameter Tuning to Common, since this steps are related to HP and NAS.

/assign @johnugeorge @gaocegege 
/cc @animeshsingh

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -17,16 +17,17 @@
...
 
-### Hyperparameter Tuning
+### Common
 
 - Delete Suggestion deployment after Experiment is finished [#1061](https://github.com/kubeflow/katib/issues/1061)
 - Save Suggestion state after deployment is deleted [#1062](https://github.com/kubeflow/katib/issues/1062)
 - Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
+- Support new Kubeflow operators (MXNet, XGBoost, MPI)
 - Add validation for algorithms (a.k.a suggestions) [#1126](https://github.com/kubeflow/katib/issues/1126)
+- Katib UI fixes and enhancements
```
[the remark this answers is not in the document]
Comment (andreyvelich) [reply ROADMAP.md:27]: Thanks, updated!

File: ROADMAP.md [hunk 2]
```diff
@@ -17,16 +17,18 @@
 
 ## Enhancements
 
-### Hyperparameter Tuning
+### Common
 
 - Delete Suggestion deployment after Experiment is finished [#1061](https://github.com/kubeflow/katib/issues/1061)
 - Save Suggestion state after deployment is deleted [#1062](https://github.com/kubeflow/katib/issues/1062)
 - Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
+- Support new Kubeflow operators (MXNet, XGBoost, MPI)
```
Comment (jlewi) [inline ROADMAP.md:25]: Can we come up with an extensible model for supporting different operators? Maybe rephrase this

* Design an extensible model for integrating with custom resources

**Comment by animeshsingh** [comment]
Also another item we discussed would be good to add to roadmap - a best case effort to investigate the alignment with concept and implementation of "experiments" and "jobs/runs" in KFP
https://github.com/kubeflow/kubeflow/issues/4955

File: ROADMAP.md (hunk 2 above)
Comment (johnugeorge) [reply ROADMAP.md:25]: +1

Comment (andreyvelich) [reply ROADMAP.md]: Agree, updated.

**Comment by andreyvelich** [comment]
> Also another item we discussed would be good to add to roadmap - a best case effort to investigate the alignment with concept and implementation of "experiments" and "jobs/runs" in KFP
> [kubeflow/kubeflow#4955](https://github.com/kubeflow/kubeflow/issues/4955)

Ok, I updated ROADMAP with this issue.

**Comment by gaocegege** [comment]
@andreyvelich Let me know if it is ready to merge.

**Comment by andreyvelich** [comment]
@animeshsingh @jlewi @johnugeorge Are you fine with these changes?
If yes, we can merge it.

**Comment by johnugeorge** [comment]
As there are no more changes, merging the PR

/approve
